Gruntfile.js

```diff
@@ -17,20 +17,18 @@ module.exports = function (grunt) {
     grunt.initConfig({
         // Project package file destination.
         pkg: grunt.file.readJSON("package.json"),
-        eslint: {
-            all: ["**/*.js"]
-        },
-        jsonlint: {
-            all: ["*.json", ".*.json", "tests/*.json"]
+        lintAll: {
+            sources: {
+                md: [ "**/*.md", "!node_modules/**/*.md"],
+                js: ["**/*.js"],
```

Comment:
Why doesn't this end up hitting js files in node_modules as presumably the negated pattern on line 22 is trying to guard against?

Reply:
I'd assume this is because of the .eslintignore file.